src/_includes/macros/time.html

```diff
@@ -0,0 +1,47 @@
+{# ==========================================================================
+
+   time.render()
+
+   ==========================================================================
+
+   Description:
+
+   Renders a time element with a standard datetime format when given:
+
+   datetime:             A date timestamp to format.
+
+   show_only (optional): Object of fields to show.
+                         Default is true for 'date', 'time', and 'timezone'.
+
+   timezone (optional):  Timezone string value. Default is 'America/New_York'.
+
+   ========================================================================== #}
+
+{% macro render(datetime,
+                show_only={'date':true, 'time':true, 'timezone':true},
+                timezone='America/New_York') %}
+{% from 'macros/util/format/datetime.html' import format_datetime as format_datetime %}
```

Comment:
Haven't we done this before? It was something like `import blah as blah` then in use `blah.thing`. Maybe I'm mistaken?